c10/core/impl/alloc_cpu.cpp

```diff
@@ -58,27 +58,29 @@ void* alloc_cpu(size_t nbytes) {
   void* data;
 #ifdef __ANDROID__
   data = memalign(gAlignment, nbytes);
+  CAFFE_ENFORCE(
+      data,
+      "DefaultCPUAllocator: not enough memory: you tried to allocate ",
+      nbytes,
+      " bytes.");
```

Comment:
These checks are moved inside the if-elifs blocks as the `else` branch already assures that `datta != nullptr` if `err == 0`.